webapp-django/crashstats/base/tests/test_utils.py

```diff
@@ -50,10 +50,11 @@ def test_SignatureStats(self):
             }
         }
         platforms = [
-            {'code': 'win', 'name': 'Windows'},
-            {'code': 'mac', 'name': 'Mac OS X'},
-            {'code': 'lin', 'name': 'Linux'},
-            {'code': 'unknown', 'name': 'Unknown'}]
+            {'short_name': 'win', 'name': 'Windows'},
+            {'short_name': 'mac', 'name': 'Mac OS X'},
+            {'short_name': 'lin', 'name': 'Linux'},
+            {'short_name': 'unknown', 'name': 'Unknown'}
```

Comment:
We switched platforms to have `name` and `short_name` rather than `name` and `code`.